bridgesync/processor.go

```diff
@@ -1014,6 +1056,32 @@ func DecodeGlobalIndex(globalIndex *big.Int) (mainnetFlag bool,
 	return
 }
 
+func (p *processor) startTransaction(ctx context.Context) (*sql.Tx, error) {
```

Comment:
```suggestion
func (p *processor) startReadOnlyTransaction(ctx context.Context) (*sql.Tx, error) {
```

Or, if you want to be more precise, leave the name, and pass the `IsReadonly` parameter to the function.

Reply:
Fixed https://github.com/agglayer/aggkit/pull/448/commits/dcb254465d3ab0e136a03de3c0426acf5b439862